conf/modules_colabfold.config

```diff
@@ -20,74 +20,70 @@ process {
+            params.use_gpu ? '--use-gpu-relax' : '',
+            params.use_amber ? '--amber' : '',
+            params.use_templates ? '--templates' : '',
+            params.host_url ? "--host-url ${params.host_url}" : ''
+        ].join(' ').trim()
+        publishDir = [
+            [
+                path: { "${params.outdir}/colabfold/${params.colabfold_server}" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+                pattern: '*.*'
+            ],
+            [
+                path: { "${params.outdir}/colabfold/${params.colabfold_server}/top_ranked_structures" },
+                mode: 'copy',
+                saveAs: { "${meta.id}.pdb" },
+                pattern: '*_relaxed_rank_001*.pdb'
             ]
-        }
+        ]
     }
 }
 
-if (params.colabfold_server == 'local') {
-    process {
-        withName: '.*:MMSEQS_.*' {
-            publishDir = [
-                enabled: false
-            ]
-        }
-        withName: 'MMSEQS_CREATEINDEX' {
-            ext.args  = '--remove-tmp-files 1'
-            ext.args2 = '*_seq.tsv'
-            publishDir = [
-                enabled: false
-            ]
-        }
-        withName: 'MMSEQS_COLABFOLDSEARCH' {
-            ext.args = { params.db_load_mode ? "--db-load-mode ${params.db_load_mode}" : '' }
-            publishDir = [
-                enabled: false
-            ]
-        }
-        withName: 'COLABFOLD_BATCH' {
-            if(params.use_gpu) { accelerator = 1 }
-            ext.args = [
-                params.use_gpu       ? '--use-gpu-relax' : '',
-                params.use_amber     ? '--amber' : '',
-                params.use_templates ? '--templates' : ''
-            ].join(' ').trim()
-            publishDir = [
-                [
-                    path: { "${params.outdir}/colabfold/${params.colabfold_server}" },
-                    mode: 'copy',
-                    saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
-                    pattern: '*.*'
-                ],
-                [
-                    path: { "${params.outdir}/colabfold/${params.colabfold_server}/top_ranked_structures" },
-                    mode: 'copy',
-                    saveAs: { "${meta.id}.pdb" },
-                    pattern: '*_relaxed_rank_001*.pdb'
-                ],
-            ]
-        }
+process {
+    withName: '.*:MMSEQS_.*' {
+        publishDir = [
+            enabled: false
+        ]
+    }
+    withName: 'MMSEQS_CREATEINDEX' {
+        ext.args  = '--compressed 1 --remove-tmp-files 1'
+        ext.args2 = '*_seq.tsv'
+        publishDir = [
+            enabled: false
+        ]
+    }
+    withName: 'MMSEQS_COLABFOLDSEARCH' {
+        ext.args = { params.db_load_mode ? "--db-load-mode ${params.db_load_mode}" : '' }
+        publishDir = [
+            enabled: false
+        ]
+    }
+    withName: 'COLABFOLD_BATCH' {
```

Comment:
I see this module is duplicated in this config file, I think this last one will override the previous configuration, which you need when you run it with `params.colabfold_server == 'webserver'`, no?

The other modules as they only appear once should be fine